cmd/create.go

```diff
@@ -125,6 +125,21 @@ func expandOrDie(path string) (config.Blueprint, *config.YamlCtx) {
 	// Expand the blueprint
 	checkErr(bp.Expand(), ctx)
 	validateMaybeDie(bp, *ctx)
+
+	// TODO: Remove this warning when v5 deprecation is complete
+	alreadyContainsV5 := false
```

Comment:
nit. We can put it into separate function to limit bloat of `expandOrDie ` e.g. `v5DeprecationWarning(bp)`